dependencies/system.py

```diff
@@ -452,12 +479,12 @@ def parseCommand(self, data):
     def pumpVolume(self, info):
         self.packets.append(f'Y') # Pump module number
         syringeType = info[3]
-        self.packets.append(f'S{syringeType}')
+        # self.packets.append(f'S{syringeType}')
         if type(info[4]) == str:
             volume = int(re.findall(r'\d+', info[4])[0])
         else:
             volume = int(info[4])
-        self.packets.append(f'm{volume}') # Pump volume
+        self.packets.append(f'S{volume}') # Pump volume
```

Comment:
Fix temporary code: The syringe pump firmware should now be flashed properly following the command structure in the readme [sID1000 rID PK2 Y Sx mz] where S is the syringe type and m is the volume to be pumped.

Reply:
Noted. I've got a big consolidation PR coming after this, so I'll roll it into that. It comes with combining multiple versions of templates on the HTML side, so it makes sense to put these two things together.